app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -218,44 +218,137 @@ var AttachmentsPanelButton = React.createClass({
     };
   },
 
+  getInitialState: function () {
+    return {
+      filter: '',
+      maxHeight: 'none'
+    };
+  },
+
+  componentDidMount: function () {
+    $(window).on('resize.attachment-filter', this.updateMaxHeight);
+    Stores.attachmentFilterStore.on('change', this.onChange);
+  },
+
+  componentWillUnmount: function () {
+    $(window).off('resize.attachment-filter', this.updateMaxHeight);
```

Comment:
I think we could probably resize the attachment list using media queries. I'm guessing you want to limit how many attachments are visible depending on the screensize. If that is the case I would prefer a css solution to this. @robertkowalski might be able to recommend an approach.